plugins/iframedialog/plugin.js

```diff
@@ -154,7 +154,7 @@ CKEDITOR.plugins.add( 'iframedialog', {
 							parentContainer = iframe.getParent(),
 							callIndex = CKEDITOR.tools.addFunction( contentLoad ),
 							html = myHtml.replace( '%1', callIndex ).replace( '%2', CKEDITOR.tools.htmlEncode( elementDefinition.src ) );
```

Comment:
What's supposed to be in `myHtml`? Seems like it's set in line 142

Reply:
It is set by calling [`uiElement`](https://sourcegraph.com/github.com/ckeditor/ckeditor4/-/blob/plugins/dialog/plugin.js?L2510).
myHtml is passed as htmlList and the only time that gets modified is on line 2582 when html is pushed to it. html is set in two steps. The first passes the attributes using htmlEncode so that should be safe. The second uses a parameter called innerHtml but we pass nothing for that in our uiElement call so it defaults to the empty string. The second is nodeName which we pass 'iframe' for. Overall it looks like this would be safe since it would end up constructed as something like `<iframe att1 = encoded_att1 att2 = encoded_att2></iframe>`